R/tutorial.R

```diff
@@ -33,6 +33,9 @@ use_tutorial <- function(name, title, open = rlang::is_interactive()) {
   use_git_ignore("*.html", directory = dir_path)
   use_dependency("learnr", "Suggests")
 
+  html_path <- path(dir_path, asciify(name), ext = ".html")
+  use_build_ignore(html_path)
+
```

Comment:
Is this not already handled by the `use_git_ignore()` call 2 lines above?

Reply:
It's my bad! I thought not including the`.html` in `.Rbuildignore` would fail the local check, but that doesn't seem to be the case. Fixing it now.

Reply:
I prefer the smallest change possible, so let's merge this. If we have more ignoring to do, we'll find out! But I think this is good to go.